packages/foliate-deb/foliate-deb.pacscript

```diff
@@ -0,0 +1,8 @@
+name="foliate-deb"
```

Comment:
Add `gives`.

Reply:
Why would `gives="com.github.johnfactotum.foliate"` not work?

Reply:
I had tested it with gives="com.github.johnfactotum.Foliate", and it didn't work.

I didn't realize that the `F` in `.Foliate` would break gives.